plugin/trino-hive/src/test/java/io/trino/plugin/hive/HiveQueryRunner.java

```diff
@@ -128,6 +130,12 @@ public SELF addHiveProperty(String key, String value)
             return self();
         }
 
+        public SELF setExchangeManagerProperties(Map<String, String> exchangeManagerProperties)
```

Comment:
can we call this method `setExchangeManager` and pass type of exchange manager explicitly. For now you can assert that it is "filesystem" as we don't have any others.

Reply:
Currently there is no method to directly set `ExchangeManager`. We only have `loadExchangeManager(String name, Map<String, String> properties)` as of now.

Reply:
Yeah I know. Just felt it would be cleaner to explicitly pass the exchangemanager name from the user of `HiveQueryRunner.Builder` to `queryRunner.loadExchangeManager()`. Not super important.